### DISCUSSION TITLE
setvalue duplicating entry

### DISCUSSION THREAD

**Discussion by Marcelogiovane** [sa-si-dev/virtual-select#147] [Q&A] [answered]
when setting setValue with an array the selection is duplicated and displayed.

**Comment by sa-si-dev** [comment]
> when setting setValue with an array the selection is duplicated and displayed.

I will check it

**Comment by sa-si-dev** [comment]
> when setting setValue with an array the selection is duplicated and displayed.

Please share the code you are using to reproduce it.

**Comment by Marcelogiovane** [comment, marked as answer]
Sorry, there was an error in the formation of the options, generating the duplicity in the definition of the value. Virtual Select is still fine.
